src/core/mentions/index.ts

```diff
@@ -21,10 +22,20 @@ export async function openMention(mention?: string): Promise<void> {
 		return
 	}
 
-	if (mention.startsWith("/")) {
+	if (
+		((osInfo === "unix" || osInfo === undefined) && mention.startsWith("/")) ||
```

Comment:
Would this work? Feels a little cleaner to handle all possible osInfo (though I'm not really sure what the other options are).

```suggestion
		(osInfo !== "win32" && mention.startsWith("/")) ||
```